src/loki/worker.cc

```diff
@@ -265,7 +264,7 @@ loki_worker_t::work(const std::list<zmq::message_t>& job,
     const auto& options = request.options();
 
     // check there is a valid action
-    if (!options.has_action() || actions.find(options.action()) == actions.cend()) {
+    if (!options.has_action()) {
```

Comment:
same as above. seems like you merged master into your branch imporoperly?

Reply:
@kevinkreiser after reverting my revert, I now fail in CI.  

> Processing triggers for dbus (1.12.2-1ubuntu1.2) ...
>  add-apt-repository ppa:valhalla-core/valhalla
>  Library, development headers and executables associated with the Valhalla open source routing project: github.com/valhalla
>  More info: https://launchpad.net/~valhalla-core/+archive/ubuntu/valhalla
> Error: retrieving gpg key timed out.

  Do you know how to fix this?

Reply:
yep thats emphemeral, looks like second run got it